src/components/TilesShared/ImageWithPlaceholder.js

```diff
@@ -22,6 +24,8 @@ const ImageWithPlaceholder = ({ src, ...rest }) => {
 
 ImageWithPlaceholder.propTypes = {
   src: PropTypes.string.isRequired,
+  width: PropTypes.number,
+  height: PropTypes.number,
```

Comment:
it's a detail, but we could make these props of type `string`, so you can define the units yourself

Reply:
Fixed